sources/web/datalab/polymer/components/datalab-files/datalab-files.ts

```diff
@@ -180,12 +180,12 @@ class FilesElement extends Polymer.Element {
   }
 
   _getNotebookUrlPrefix() {
-    let prefix = location.protocol + '//' + location.host + '/';
+    let prefix = location.protocol + '//' + location.host + ApiManager.basepath + '/';
```

Comment:
Done.  As discussed offline: only datalab-files needs the basepath because the notebooks/files are stored on the VM.